developer_guide.md

```diff
@@ -0,0 +1,100 @@
+# Development guidelines for KubeSlice Worker
+
+The KubeSlice-worker operator manages the life cycle of KubeSlice worker cluster related [CRDs](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/).
+It is strongly recommended to use a released version. Follow the instructions provided in this [guide](https://docs.avesha.io/opensource/registering-the-worker-cluster#Bookmark162).
+
+## Building and Installing `kubeslice-worker` in a Local Kind Cluster
+For more information, see [getting started with kind clusters](https://docs.avesha.io/opensource/getting-started-with-kind-clusters).
+
+### Setting up Development Environment
+
+* Go (version 1.17 or later) installed and configured in your machine ([Installing Go](https://go.dev/dl/))
+* Docker installed and running in your local machine
+* A running [`kind`](https://kind.sigs.k8s.io/)  cluster
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed and configured
+* Follow the getting started from above, to install [kubeslice-controller](https://github.com/kubeslice/kubeslice-controller) 
+
+
+
+### Building Docker Images
+
+1. Clone the latest version of kubeslice-controller from  the `master` branch.
+
+```bash
+git clone https://github.com/kubeslice/worker-operator.git
+cd worker-operator
+```
+
+2. Adjust image name variable `IMG` in the [`Makefile`](Makefile) to change the docker tag to be built.
+   Default image is set as `IMG ?= aveshasystems/worker-operator:latest`. Modify this if required.
+
+```bash
+make docker-build
+```
+
+3. Loading kubeslice-worker Image Into Your Kind Cluster ([`link`](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster))
```

Comment:
Change `Image Into Your Kind Cluster` to `image into your Kind cluster` 

and end this sentence with a period/full stop